adding FAQ for antivirus coverage (#96100)
Azure/azure-content: articles/defender-for-cloud/integration-defender-for-endpoint.md

### What are the licensing requirements for Microsoft Defender for Endpoint?
Defender for Endpoint is included at no extra cost with **Microsoft Defender for Servers**. Alternatively, it can be purchased separately for 50 machines or more.

### Do I need to buy a separate antivirus solution to protect my machines?

> No. With MDE integration in Defender for Servers, you will also get antimalware protection on your machines.

Reviewer: ```suggestion
No. With MDE integration in Defender for Servers, you'll also get antimalware protection on your machines.
```

> No. With MDE integration in Defender for Servers, you will also get antimalware protection on your machines.

Reviewer: The answer, which mentions antimalware, doesn't seem to match the question, which mentions antivirus. What's the difference between them? Can you clarify so that the answer seems to more directly answer the question?

I would expect something like: MDE integration in Defender for Servers provides antivirus and antimalware protection for your machines.

Author: I changed the FAQ so it now talks about anti-malware instead of antivirus. Anti-malware is the more modern approach, covering not only viruses, but also Trojans, Worms, and other malicious software.
